includes/amp-helper-functions.php

```diff
@@ -1608,6 +1608,12 @@ function amp_get_content_sanitizers( $post = null ) {
 		);
 	}
 
+	$is_auto_lightbox_disabled = apply_filters( 'amp_auto_lightbox_disabled', false );
```

Comment:
I'm late to respond to https://github.com/ampproject/amp-wp/issues/5122#issuecomment-1050609393.

After thinking about it some more, I think it's probably good to indeed disable by default. Otherwise, users who see the Lightbox toggle will be confused what purpose it serves if the images are lightboxed anyway.